gradle.properties

```diff
@@ -13,6 +13,8 @@ evosuiteVersion = 1.0.5
 pluginSinceBuild = 241
 pluginUntilBuild = 241.*
 
+ideVersionVerifier = IC-2024.1
+
```

Comment:
How is this variable used?

Reply:
It does not, thank you! I will delete it)